pkg/sqlcmd/sqlcmd_test.go

```diff
@@ -109,6 +109,28 @@ func TestSqlCmdQueryAndExit(t *testing.T) {
 	}
 }
 
+func TestSqlCmdOutputAndError(t *testing.T) {
+	s, outfile, errfile := setupSqlcmdWithFileErrorOutput(t)

```

Comment:
don't you need to test the s.IncludeFile path too?

Reply:
Added test for s.IncludeFile